### ISSUE TITLE
RemoteServiceException: Context.startForegroundService() did not then call Service.startForeground(): ServiceRecord{df282...

### ISSUE THREAD

**Issue by pocmo** [mozilla-mobile/fenix#4853]
https://sentry.prod.mozaws.net/operations/fenix-nightly/issues/6158109/

```
RemoteServiceException: Context.startForegroundService() did not then call Service.startForeground(): ServiceRecord{df2820b u0 org.mozilla.fenix/mozilla.components.feature.media.service.MediaService}
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1872)
    at android.os.Handler.dispatchMessage(Handler.java:106)
    at android.os.Looper.loop(Looper.java:214)
    at android.app.ActivityThread.main(ActivityThread.java:7050)
    at java.lang.reflect.Method.invoke(Method.java)
...
(2 additional frame(s) were not displayed)

Context.startForegroundService() did not then call Service.startForeground(): ServiceRecord{df2820b u0 org.mozilla.fenix/mozilla.components.feature.media.service.MediaService}
```

**Comment by pocmo** [comment]
Needs fix in AC: https://github.com/mozilla-mobile/android-components/issues/4172

**Comment by pocmo** [comment]
Feature is still behind Nightly-only flag!

**Comment by pocmo** [comment]
Fixed with the next AC 10.0.0-SNAPSHOT

**Comment by pocmo** [comment]
Crashes are gone.
